docs/src/.vuepress/config.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
     ],
     ["meta", { name: "og:type", content: "website" }],
     ["meta", { name: "og:url", content: "https://discordeno.mod.land" }],
+    ["meta", { name: "og:image", content: "/logo.png" }],
```

Comment:
```suggestion
    ["meta", { name: "og:image", content: "/favicon.png" }],
```
*Make sure to delete the *logo.png* file from *public/* directory.

Reply:
That are two different images. The favicon is 128x128 and the logo is 300x300